app/konnect/updates.md

```diff
@@ -9,70 +9,65 @@ an application that lets you manage configuration for multiple runtimes
 from a single, cloud-based control plane, and provides a catalog of all deployed
 services. [Try it today!](https://cloud.konghq.com/quick-start)
 
-## November 2022
+## December 2022
```

Comment:
I thought we decided to do the changelog additions at the end of the month vs rolling, because I think these would still cause merge conflicts.

Reply:
That was my recollection as well, @cloudjumpercat - could you pull this off the PR and we can start a new one for 12/22 changelog?

Reply:
Fixed with @Guaris help :)